backup/src/main/scala/batches/Backup.scala

```diff
@@ -0,0 +1,70 @@
+package batches
+
+import java.util
+import javax.inject.Inject
+
+import com.microsoft.azure.storage.CloudStorageAccount
+import com.microsoft.azure.storage.blob.SharedAccessBlobPermissions._
+import com.microsoft.azure.storage.blob._
+import gu.msnotifications.{NotificationHubJobType, _}
+import org.joda.time.{DateTime, DateTimeZone}
+import play.api.libs.ws.WSClient
+import play.api.Logger
+import collection.JavaConversions._
+
+import scala.concurrent.{ExecutionContext, Future}
+import scalaz.{-\/, \/-}
+
+class Backup @Inject() (conf: BackupConfiguration, ws: WSClient)(implicit ec: ExecutionContext) extends Batch {
```

Comment:
All the Microsoft classes are final, in my opinopn the overhead of creating classes that delegates to Microsoft's code to allow testing doesn't justify creating a unit test here.
I'm opening it for debate though, if anyone has a very string opinion against it.